src/test/java/io/aiven/kafka/connect/gcs/config/GcsSinkConfigTest.java

```diff
@@ -16,13 +16,16 @@
 
 package io.aiven.kafka.connect.gcs.config;
 
+import static io.aiven.kafka.connect.gcs.GcsSinkConfig.*;
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertIterableEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
 
 import java.io.IOException;
 import java.net.URL;
```

Comment:
nit: not used any more as for the removed code.

Reply:
Removed (along with all other pmd probmels)